src/util/Util.js

```diff
@@ -103,13 +103,14 @@ define(['okta', 'util/Logger'], function (Okta, Logger) {
 
   // Trigger an afterError event
   Util.triggerAfterError = function (controller, err) {
-    if (!err.statusCode) {
+    if (err.xhr && err.xhr.status) {
```

Comment:
Changing this logic to only include the `statusCode` when present, instead of possibly setting this value to `undefined`.